tests/admin/import/test-class-import-aioseo.php

```diff
@@ -24,68 +24,132 @@ public function setUp() {
 	}
 
 	/**
-	 * @covers WPSEO_Import_AIOSEO::plugin_name
+	 * @covers WPSEO_Import_AIOSEO::get_plugin_name
```

Comment:
Test methods also need a short description - goes for all test methods... :persevere: